sw/Modules/Comms/Src/can_sensors.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * can_sensors.c
+ *
+ *  Created on: Mar 19, 2025
+ *      Author: Hardy Yu
+ */
+
+#include "can_sensors.h"
+
+extern CanardInstance canard;
+
+void
+convertImuToDroneCAN (const ImuData_S *src,
```

Comment:
Similar comment to above. 

Maybe I should throw in a clang config so that our files can all be autoformatted to be the same "way"